docs/src/user-docs/reference-telemetry.md

```
@@ -35,6 +35,31 @@ filtering, redaction, aggregation, and forwarding. For a matching baseline, star
 (Helm).
 :::
 
+## Configuration Settings
+
+You can configure the interval at which metrics are exported for each instrumented component. The `telemetry_update_interval_ms` setting allows you to control the export frequency (in milliseconds) and defaults to `60000` (60 seconds).
+
+::::{tab-set}
+:::{tab-item} Docker Compose
+:sync: docker
+Edit `clp-config.yaml` to set the interval per component:
+```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)